packages/catalog-model/examples/acme/backstage-group.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: backstage.io/v1alpha1
+kind: Group
+metadata:
+  name: backstage
+  description: The backstage business unit
```

Comment:
the texts still say business unit :) but no big deal

Reply:
I fixed this in the next commit.